restapi/src/main/java/gov/cdc/sdp/cbr/restapi/InputController.java

```diff
@@ -21,52 +22,69 @@
 import com.google.gson.Gson;
 import com.google.gson.JsonSyntaxException;
 
+import gov.cdc.sdp.cbr.trace.TraceService;
+import gov.cdc.sdp.cbr.trace.model.TraceStatus;
+
 @Controller
 public class InputController {
 
     @Autowired
     protected CamelContext camelContext;
     
+    @Autowired
+    private TraceService traceService;
+    
     @Value("${input.post.endpoint}")
     public String endpoint;
     
     @RequestMapping(value="/cbr/input", method = RequestMethod.POST)
     
-    public @ResponseBody String doSomething(
+    public @ResponseBody String processInputFile (
                 @RequestParam("id") String id,
                 @RequestParam("source") String source,
+                // TODO: Add createdAt, change createdAt in transformer to receivedAt?
                 @RequestParam("metadata") String jsonMetadata,
-                @RequestParam("file") MultipartFile file)  {
+                @RequestParam("file") MultipartFile file) throws Exception  {
+        
         String cbrId = "CBR_" + source + "_" + id;
         
-        // TODO: Log message in
+        try {
+			traceService.addTraceMessage(cbrId, source, TraceStatus.INFO, "Message from id " + id + " received");
+		} catch (SQLException e) {
+			traceService.addTraceMessage(cbrId, source, TraceStatus.ERROR, e.getMessage());
```

Comment:
I know that I asked in the last review to do more with the exception than just printing the stack trace. I can see how logging it would be good, but I don't think that works here. The code is catching a SQLException generated by the log service, so attempting to log again doesn't seem like it would work. It might be best to get rid of the try/catch block used originally and just throw the exception from the method.